compiler/src/iree/compiler/Dialect/Encoding/IR/test/roundtrip.mlir

```diff
@@ -223,11 +223,11 @@ func.func @testing_without_layouts(%arg0: tensor<?x?xf32, #encoding>) -> tensor<
 
 // -----
 
-#encoding = #iree_encoding.testing<[#iree_encoding.specialization_resolver<123>]>
+#encoding = #iree_encoding.testing<layouts = [#iree_encoding.specialization_resolver<123>]>
```

Comment:
huh, I did not catch this when I improved the testing encodings. It is better to use `iree_encoding.specialized<123>` instead.